pyomo/gdp/plugins/binary_multiplication.py

```diff
@@ -0,0 +1,175 @@
+from .gdp_to_mip_transformation import GDP_to_MIP_Transformation
+from pyomo.common.config import ConfigDict, ConfigValue
+from pyomo.core.base import TransformationFactory
+from pyomo.core.util import target_list
+from pyomo.gdp import Disjunction
+from pyomo.gdp.plugins.gdp_to_mip_transformation import GDP_to_MIP_Transformation
+from pyomo.core.util import target_list
+from weakref import ref as weakref_ref
+import logging
+
+
+logger = logging.getLogger('pyomo.gdp.binary_multiplication')
```

Comment:
I'm starting to like the convention of setting this as `logging.getLogger(__name__)`.